docs/src/main/sphinx/connector/hive.rst

```diff
@@ -521,6 +521,36 @@ Property Name                                                   Description
 
 ``hive.metastore.thrift.client.ssl.trust-certificate-password`` Password for the trust store
 
+``hive.metastore.thrift.delete-files-on-drop``                  When a table is dropped, delete the underlying files
```

Comment:
@posulliv FYI

Reply:
Just to be clear that this only applies to managed tables, perhaps change this sentence:

> When a table is dropped, delete the underlying files corresponding to the table

to:

> When a managed table is dropped, ensure Trino deletes all data for the table.